yjit/src/disasm.rs

```diff
@@ -195,6 +198,73 @@ pub fn disasm_addr_range(cb: &CodeBlock, start_addr: usize, end_addr: usize) ->
     return out;
 }
 
+/// Macro useful for raw string literals
```

Comment:
Should explain in the comment what this macro does.

Reply:
Ah yes, I forgot updating this https://github.com/ruby/ruby/pull/7470/commits/86999bb77300dfb1bbc591db5e7d082bf6ff6ade. Thanks for noticing it.